client/src/controllers/scores/score_controller.js

```diff
@@ -90,13 +90,30 @@ class ScoreController {
 		}
 
 		this.Scores.update(this.data._id, updateData)
-		.then(() => {
-			self.$scope.$emit('reload')
-		}).catch(() => {
-			self.Notifications.error('Unable to update score: Possible network error.')
-		})
+			.then(() => {
+				self.$scope.$emit('reload')
+			}).catch(() => {
+				self.Notifications.error('Unable to update score: Possible network error.')
+			})
+	}
+
+	teamRoundError() {

```

Comment:
These functions could just return a one line:
```javascript
teamRoundError() {
    return !this.loading && this.data.round == null
}
```
Also notice that there is no use for the `self` variable, so it shouldn't exist

Reply:
Just checked and they give an error without the self variable, even in teamRoundError() where it is totally unused. No idea why.

Reply:
Just ran it localy and it works perfectly fine... what error do tou get?